torch/testing/_internal/common_methods_invocations.py

```diff
@@ -5978,6 +5997,9 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
         numel = torch.tensor(t.shape)[kwargs.get('dim')].prod()
         yield ((), {'correction': numel // 2})
 
+def ref_pairwise_distance(input1, input2):
+    pass
+
```

Comment:
Oops, that shouldn't be here. I've already removed locally and will push this with the next stack update.